integration_tests/test_aggregator_batch_commands.py

```diff
@@ -137,12 +136,12 @@ def on_market_cycle(self, market_info):
 
                     # The bids have been issued in the correct order
                     assert [
-                        bid['original_bid_price'] for bid in issued_bids
-                    ] == [27, 28, 29, 30]
+                               bid['original_bid_price'] for bid in issued_bids
+                           ] == [27, 28, 29, 30]
 
                     # The only two bids left are the last ones that have been issued
                     assert [bid['id'] for bid in current_bids] == \
-                        [bid['id'] for bid in issued_bids[-2:]]
+                           [bid['id'] for bid in issued_bids[-2:]]
```

Comment:
Same question about indentation here. I think we can safely follow PEP8 in these cases.
Also regarding backslashes (see this comment: https://github.com/gridsingularity/d3a-api-client/pull/94#discussion_r601273204)

Reply:
Update: In this case, PEP8 actually states that backslashes are ok with `assert` statements:
> Another such case is with assert statements